brainstormforce/ultimate-addons-for-gutenberg
# Fix: Taxonomy List - issues in `next-release`

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/3sAgbH1y7qY/6339e334-d7a9-4397-b2ab-2aac691488a7?redirect=true&view=6e170ea7-8708-4546-8b82-23d2eebdf694&t=attributes
- Fix Border not applying issue.
- Move Bottom spacing in Title instead of changing the name in spacing control.

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop Taxonomy block.
- Add a border, apply color, width, and styling to check it works in the editor/frontend.
- Check block on responsive devices in editor is it look good or not.

### Checklist:
